### PR TITLE
Added personal signature system

### LINKED ISSUE #364: personal signatures
Need a mechanic where any player can sign a piece of paper with their signature. This should work like a stamp: a nice inscription with the player's full name, with a prohibition to edit anything on the paper further.
Ideally it should be a nice italic font, but a standard staple visual will do for MVPs as well

необходима механика, когда любой игрок может подписать лист бумаги своей подписью. Это должно работать как штамп: красивая надпись с полным именем игрока, с запретом что-либо редактировать на бумаге далее.
в идеале это должен быть красивый курсивный шрифт, но для MVP подойдет и стандартный визуал штапма

### PR DESCRIPTION
**Pull request by Tornado-Technology**
## About the PR
Reed #364, I'm lazy

## Why / Balance
Resolves #364

## Media

https://github.com/user-attachments/assets/0a7a66df-34e5-4938-8cf9-ae580c643a7c

### CODE REVIEW

File: Content.Server/_CP14/PersonalSignature/CP14PersonalSignatureSystem.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+        if (!CanSign(args.Using))
+            return;
+
+        if (HasSign(entity, mind.CharacterName))
+            return;
+
+        args.Verbs.Add(new AlternativeVerb
+        {
+            Text = "Sign",
```
Comment (TheShuEd) [inline Content.Server/_CP14/PersonalSignature/CP14PersonalSignatureSystem.cs:36]: Loc required

File: Content.Server/_CP14/PersonalSignature/CP14PersonalSignatureSystem.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    private void Sign(Entity<PaperComponent> target, string name)
+    {
+        var info = new StampDisplayInfo
+        {
+            StampedName = name,
+            StampedColor = Color.Gray,
+        };
+
```
Comment (TheShuEd) [inline Content.Server/_CP14/PersonalSignature/CP14PersonalSignatureSystem.cs:73]: add some audio telegraphy

**Review by TheShuEd — CHANGES_REQUESTED** [review]

File: Content.Server/_CP14/PersonalSignature/CP14PersonalSignatureSystem.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    private void Sign(Entity<PaperComponent> target, string name, SoundSpecifier? sound)
+    {
+        var info = new StampDisplayInfo
+        {
+            StampedName = name,
+            StampedColor = Color.Gray,
+        };
+
+        if (sound is not null)
+            _audio.PlayEntity(sound, Filter.Pvs(target), target, true);
```
Comment (TheShuEd) [inline Content.Server/_CP14/PersonalSignature/CP14PersonalSignatureSystem.cs:75]: PlayPvs? No?

Comment (Tornado-Technology) [reply Content.Server/_CP14/PersonalSignature/CP14PersonalSignatureSystem.cs:75]: That's same, but play entity, attached sounds to entity

**Review by TheShuEd — APPROVED** [review]

**Pull request merged**